### PR TITLE
[FX-4415] document `external_ticket_references` route

### PR DESCRIPTION
**Pull request by kittaakos**
[FX-4415](https://zombie.atlassian.net/browse/FX-4415)

Before:
Screenshot 2024-07-01 at 16 12 07

After:
Screenshot 2024-07-01 at 16 11 41

[FX-4415]: https://zombie.atlassian.net/browse/FX-4415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: versions/v2/content/external-ticket-references.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+  -H "X-Org-Token: YOUR-V2-ORGANIZATION-TOKEN"
+  --data '{"findings":[{"id":"vl_JTovzf8AW1afCRpKJejse3"},{"id":"dfi_2bZrrKE3Hay7oLrbNUponW"}]}'
+```
+
+### HTTP Request
+
+`POST https://api.us.cobalt.io/external_ticket_references/search`
+
+### Body
```
[the remark this answers is not in the document]
Comment (mahmoudadly-cobalt) [reply versions/v2/content/external-ticket-references.md:77]: Alternatively, have a smaller heading if you wish to nest this section under `### HTTP Request`

[the remark this answers is not in the document]
Comment (kittaakos) [reply versions/v2/content/external-ticket-references.md:77]: Looks like I got one of the wrong examples: https://github.com/search?q=repo%3Acobalthq%2Fcobalt-api-docs%20%22%23%23%20Body%22&type=code

File: versions/v2/content/external-ticket-references.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+### HTTP Request
+
+`POST https://api.us.cobalt.io/external_ticket_references`
+
+### Request Body
+
+| Field              | Description                                                                   |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `title`            | A short descriptive title of the external ticket. For example, the ticket ID. |
+| `ticketing_system` | One of the supported [ticking system](#ticketing-system).                     |
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:126]:
```suggestion
| `ticketing_system` | One of the supported [ticketing system](#ticketing-system).                     |
```

File: versions/v2/content/external-ticket-references.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+### Response
+
+You get a `201` response code for a successful request.
+
+| Field              | Description                                                                   |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `id`               | A unique ID representing the external ticket reference. Starts with `efr_`    |
+| `title`            | A short descriptive title of the external ticket. For example, the ticket ID. |
+| `ticketing_system` | One of the supported [ticking system](#ticketing-system).                     |
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:139]:
```suggestion
| `ticketing_system` | One of the supported [ticketing system](#ticketing-system).                     |
```

File: versions/v2/content/external-ticket-references.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+> The above command returns no data and a `204` response code when successful.
+
+This endpoint deletes the external ticket reference.
+
+<aside class="notice">
+Remember - you can only request external ticket reference scoped to the organization specified in the
+<code>X-Org-Token</code> header.
+</aside>
+
+### Ticketing System
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:162]: I don't I understand this section. What is it and why is it nested under `## Delete an External Ticket Reference`?

Comment (kittaakos) [reply versions/v2/content/external-ticket-references.md]: Now, it's `<h2>`. Thanks for catching this too!

https://github.com/cobalthq/cobalt-api-docs/pull/245/commits/cd154c75af98c5e23d376ce42ebb179097360299

File: versions/v2/content/external-ticket-references.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+> The above command returns no data and a `204` response code when successful.
+
+This endpoint deletes the external ticket reference.
+
+<aside class="notice">
+Remember - you can only request external ticket reference scoped to the organization specified in the
+<code>X-Org-Token</code> header.
+</aside>
+
+## Ticketing System
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:162]:
```suggestion
## Supported Ticketing Systems
```

File: versions/v2/content/external-ticket-references.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+### Response
+
+You get a `201` response code for a successful request.
+
+| Field              | Description                                                                   |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `id`               | A unique ID representing the external ticket reference. Starts with `efr_`    |
+| `title`            | A short descriptive title of the external ticket. For example, the ticket ID. |
+| `ticketing_system` | One of the supported [ticketing system](#ticketing-system).                     |
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:139]:
```suggestion
| `ticketing_system` | One of the [supported ticketing systems](#supported-ticketing-systems).                     |
```

File: versions/v2/content/external-ticket-references.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+### HTTP Request
+
+`POST https://api.us.cobalt.io/external_ticket_references`
+
+### Request Body
+
+| Field              | Description                                                                   |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `title`            | A short descriptive title of the external ticket. For example, the ticket ID. |
+| `ticketing_system` | One of the supported [ticketing system](#ticketing-system).                     |
```
Comment (mahmoudadly-cobalt) [inline versions/v2/content/external-ticket-references.md:126]:
```suggestion
| `ticketing_system` | One of the [supported ticketing systems](#supported-ticketing-systems).                     |
```

**Review by mahmoudadly-cobalt — COMMENTED** [review]
Sorry I did not catch this earlier. This is my last set of notes. 
I need to leave soon after posting this, but please get an approval and go with it without my approval.

**Pull request merged**